.editorconfig

```diff
@@ -15,3 +15,6 @@ trim_trailing_whitespace = true
 [*.{bat,cmd,[Bb][Aa][Tt],[Cc][Mm][Dd]]
 # DOS/Win *requires* BAT/CMD files to have CRLF newlines
 end_of_line = crlf
+
+[*.{yml, yaml}]
```

Comment:
This change isn't required an could be removed

Reply:
Repository contains appveyor yaml file, with editor config set to * size 4. Appveyor file was indented with 2 spaces (which is correct and should be) == Neede for editors to catch up or reformat appveyor to 4 spaces.

Reply:
You're right 😄